packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx

```diff
@@ -360,6 +365,26 @@ export const Sep6Deposit = () => {
     );
   }
 
+  if (sep6DepositAsset.status === ActionStatus.KYC_DONE) {
+    return (
+      <Modal
+        visible={isInfoModalVisible}
+        onClose={() => setIsInfoModalVisible(false)}
+        parentId={CSS_MODAL_PARENT_ID}
+      >
+        <Modal.Heading>SEP-6 Deposit</Modal.Heading>
+
+        <Modal.Body>
+          <p>Submit the deposit.</p>
+        </Modal.Body>
+
+        <Modal.Footer>
+          <Button onClick={() => dispatch(sep6DepositAction())}>Submit</Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
```

Comment:
We need this extra step to trigger transaction polling after the SEP-12 fields are submitted. We can change the copy of the modal, and the same goes for withdrawal.

Reply:
Will this be added in a different PR?

Reply:
This is already implemented. It looks like it's conflicting with the optional `instructions` for deposit. I'm looking into that.

Reply:
This is an issue with AP's implementation. The transaction moves into the completed status before the wallet polls and sees the `pending_user_transfer_start` status.